src/styles/customCSS.ts

```diff
@@ -1,5 +1,9 @@
 import { CSSProperties } from 'react'
 
+export const activeStyle: CSSProperties = {
+  backgroundColor: 'rgba(0,0,0,.25)',
```

Comment:
it seems to be a more complex story around this. Button could be of `primary` or `secondary` type - and for these types its visual look should differ. Currently it seems that only the case of default button is handled.

Reply:
Ah, I didn't think of that. Let me work on it.

Reply:
@kuzhelov I made some changes. Please check.